clients/benchmarks/client.cpp

```diff
@@ -36,272 +37,22 @@
-          po::value<double>(&argus.alpha)->default_value(1.0), "specifies the scalar alpha")
-        
-        ("beta",
-         po::value<double>(&argus.beta)->default_value(0.0), "specifies the scalar beta")
-              
-        ("function,f",
-         po::value<std::string>(&function)->default_value("gemv"),
-         "BLAS function to test. Options: gemv, ger, syr, trsm, trmm, symv, syrk, syr2k")
-        
-        ("precision,r", 
-         po::value<char>(&precision)->default_value('s'), "Options: h,s,d,c,z")
-        
-        ("a_type", 
-         po::value<char>(&a_type)->default_value('s'), "Options: h,s,d,c,z"
-         "Precision of matrix A, only applicable to BLAS_EX")
-        
-        ("b_type", 
-         po::value<char>(&b_type)->default_value('s'), "Options: h,s,d,c,z"
-         "Precision of matrix B, only applicable to BLAS_EX")
-        
-        ("c_type", 
-         po::value<char>(&c_type)->default_value('s'), "Options: h,s,d,c,z"
-         "Precision of matrix C, only applicable to BLAS_EX")
-        
-        ("d_type", 
-         po::value<char>(&d_type)->default_value('s'), "Options: h,s,d,c,z"
-         "Precision of matrix D, only applicable to BLAS_EX")
-        
-        ("compute_type", 
-         po::value<char>(&compute_type)->default_value('s'), "Options: h,s,d,c,z"
-         "Precision of computation, only applicable to BLAS_EX")
-        
-        ("transposeA",
-         po::value<char>(&argus.transA_option)->default_value('N'),
-         "N = no transpose, T = transpose, C = conjugate transpose")
-        
-        ("transposeB",
-         po::value<char>(&argus.transB_option)->default_value('N'),
-         "N = no transpose, T = transpose, C = conjugate transpose")
-        
-        ("side",
-         po::value<char>(&argus.side_option)->default_value('L'),
-         "L = left, R = right. Only applicable to certain routines")
-        
-        ("uplo",
-         po::value<char>(&argus.uplo_option)->default_value('U'),
-         "U = upper, L = lower. Only applicable to certain routines") // xsymv xsyrk xsyr2k xtrsm
-                                                                     // xtrmm
-        ("diag",
-         po::value<char>(&argus.diag_option)->default_value('N'),
-         "U = unit diagonal, N = non unit diagonal. Only applicable to certain routines") // xtrsm
-                                                                                          // xtrmm
-        ("batch",
-         po::value<rocblas_int>(&argus.batch_count)->default_value(1),
-         "Number of matrices. Only applicable to batched routines") // xtrsm xtrmm xgemm
-
-        ("verify,v",
-         po::value<rocblas_int>(&argus.norm_check)->default_value(0),
-         "Validate GPU results with CPU? 0 = No, 1 = Yes (default: No)")
-
-        ("iters,i",
-         po::value<rocblas_int>(&argus.iters)->default_value(10),
-         "Iterations to run inside timing loop")
-
-
-
-
-        ("algo",
```

Comment:
These 4 values were added since the refactoring started, so they are merged here.